Sources/ArgumentParser/Utilities/SequenceExtensions.swift

```diff
@@ -9,14 +9,9 @@
 //
 //===----------------------------------------------------------------------===//
 
-extension Sequence where Element: Equatable {
-  func uniquified() -> [Element] {
-    var sequence = Array<Element>()
-    for element in self {
-      if !sequence.contains(element) {
-        sequence.append(element)
-      }
-    }
-    return sequence
+extension Sequence where Iterator.Element: Hashable {
+  func uniquified() -> [Iterator.Element] {
+    var seen: [Iterator.Element: Bool] = [:]
```

Comment:
Would you mind using a `Set<Element>` instead of a dictionary here?

Reply:
@natecook1000 thanks!  The `updateValue` member used to make this possible is on the `Dictionary` type: https://developer.apple.com/documentation/swift/dictionary/3127179-updatevalue.  Is there an alternative provided by `Set`?

Reply:
You can use [`insert(_:)`](https://developer.apple.com/documentation/swift/set/3128848-insert), which returns a tuple — the first element of the tuple is `true` if the element is inserted and `false` if it's already in the set.

Reply:
@natecook1000 thats awesome! I made this change